products/llbuildSwift/CMakeLists.txt

```diff
@@ -1,3 +1,37 @@
+# Returns the current architecture name in a variable
```

Comment:
Why replicate this rather than from `SwiftSupport.cmake` ?

Reply:
LLBUILD doesn't have a `SwiftSupport.cmake` yet, but we should probably pull that from somewhere.
@compnerd is there a canonical version somewhere? The one in swift-system is a few lines longer from a quick glance, so probably handles more architectures; https://github.com/apple/swift-system/blob/main/cmake/modules/SwiftSupport.cmake.

Reply:
> The one in swift-system is a few lines longer from a quick glance, so probably handles more architectures;

I just checked and the only difference is that that one doesn't handle `wasm32`, whereas this one is missing the all-caps `ARM64`. Not particularly important, can always add it later if wanted, as I'd like to get this in with the other pulls before the 5.9 branch.